avocado/core/runners/avocado_instrumented.py

```diff
@@ -111,7 +111,7 @@ def _run_avocado(runnable, queue):
                              'modulePath': module_path,
                              'params': (TreeNode(), []),
                              'tags': runnable.tags,
-                             'run.results_dir': tempfile.mkdtemp(),
+                             'run.results_dir': tempfile.mkdtemp(prefix=".avocado-task"),
```

Comment:
Your commit message says the prefix is "avocado_", but here you are using a different prefix.